x-pack/plugins/ml/common/util/anomaly_utils.js

```diff
@@ -81,13 +104,21 @@ export function getSeverityColor(normalizedScore) {
 // which ranges from -5 to +5.
 export function getMultiBucketImpactLabel(multiBucketImpact) {
   if (multiBucketImpact >= MULTI_BUCKET_IMPACT.HIGH) {
-    return 'high';
+    return i18n.translate('xpack.ml.anomaliesTable.multiBucketImpact.highLabel', {
```

Comment:
As above, these IDs should start with `xpack.ml.anomalyUtils` not `xpack.ml.anomaliesTable`